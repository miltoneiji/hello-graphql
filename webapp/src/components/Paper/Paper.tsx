import React from "react";
import styled from "styled-components";

const Paper = styled.div`
  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.18);
  padding: 32px;
`;

export default Paper;
