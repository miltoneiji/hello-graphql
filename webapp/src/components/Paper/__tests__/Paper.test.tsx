import React from "react";
import { render } from "react-testing-library";
import "jest-styled-components";

import Paper from "../Paper";

describe("<Paper />", () => {
  it("renders properly", () => {
    const { container } = render(<Paper>Hello, paper!</Paper>);

    expect(container.firstChild).toMatchSnapshot();
  });
});
