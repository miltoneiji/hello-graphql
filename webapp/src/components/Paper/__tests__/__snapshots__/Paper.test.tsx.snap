// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Paper /> renders properly 1`] = `
.c0 {
  box-shadow: 0px 2px 4px rgba(0,0,0,0.18);
  padding: 32px;
}

<div
  class="c0"
>
  Hello, paper!
</div>
`;
